src/tools/DevGenerators/SubtypesFactoryGenerator.cs

```diff
@@ -12,7 +12,7 @@ namespace Avalonia.SourceGenerator
     public class SubtypesFactoryGenerator : IIncrementalGenerator
     {
         private record struct MethodTarget(IMethodSymbol Method, string MethodDecl, ITypeSymbol BaseType, string Namespace);
-        private static readonly string s_attributeName = typeof(SubtypesFactoryAttribute).FullName;
+        private const string AttributeName = "Avalonia.SourceGenerator.SubtypesFactoryAttribute";
```

Comment:
I'm not sure if including the attribute code as a a string makes it easier to maintain.

@grokys What is our policy regarding that?